src/serialization/preprocessor.cpp

```diff
@@ -1075,8 +1075,10 @@ bool preprocessor_data::get_chunk()
 				size_t nb_arg = strings_.size() - token.stack_pos - 1;
 				if (nb_arg != val.arguments.size())
 				{
+					std::vector< std::string > locations = utils::quoted_split(val.location, ' ');
```

Comment:
Since the ``locations`` variable is never modified afterwards, this line would benefit from declaring it as a const reference. We also generally don't use spaces around the template param list except where absolutely required by C++03 (namely ``foo<bar<baz> >``).

Note that this is just a small bit of coding style advice to keep in mind for future patches and you don't need to change it in your PR yourself.